PowerVCF.psm1

```diff
@@ -4270,18 +4270,37 @@ Function Get-VCFSystemPrecheckTask {
 
         .PARAMETER id
         Specifies the unique ID of the system level precheck task.
+
+        .PARAMETER failedonly
+        Optional: only show failed subtasks
+      
     #>
 
     Param (
-        [Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$id
+        [Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$id,
+        [Parameter (Mandatory = $false)] [ValidateNotNullOrEmpty()] [Switch]$failedonly
     )
 
     Try {
         createHeader # Set the Accept and Authorization headers.
         checkVCFToken # Validate the access token and refresh, if necessary.
         $uri = "https://$sddcManager/v1/system/prechecks/tasks/$id"
-        $response = Invoke-RestMethod -Method GET -Uri $uri -Headers $headers -ContentType 'application/json'
-        $response
+    
+        Do {
+            # Keep checking until status is not IN_PROGRESS
+            $response = Invoke-RestMethod -Method GET -Uri $uri -Headers $headers -ContentType 'application/json'
```

Comment:
```suggestion
            $response = Invoke-RestMethod -Method GET -Uri $uri -Headers $headers -ContentType 'application/json'
            Start-Sleep -Seconds 2
```

Added `Start-Sleep -Seconds 2`` command inside the loop to pause for 2 seconds between each request.